web/templates/invite_user_modal.hbs

```diff
@@ -67,33 +67,20 @@
         <div id="streams_to_add">
             {{#if show_select_default_streams_option}}
             <div class="select_default_streams new-style">
-                <label class="checkbox display-block">
+                <label class="checkbox display-block modal-field-label">
```

Comment:
Why have we added the `modal-field-label` class?

Reply:
To maintain consistent margins between the labels and input fields in the modal, and as suggested by this CZO [comment](https://chat.zulip.org/#narrow/stream/101-design/topic/User.20invite.3B.20streams.20they.20should.20join/near/1724788).